astroid/_ast.py

```diff
@@ -51,16 +50,14 @@ def parse_function_type_comment(type_comment: str) -> Optional[FunctionType]:
     if _ast_py3 is None:
         return None
 
-    func_type = _ast_py3.parse(type_comment, "<type_comment>", "func_type")
+    func_type = _ast_py3.parse(type_comment, "<type_comment>", "func_type")  # type: ignore[attr-defined]
```

Comment:
```suggestion
    func_type = _ast_py3.parse(type_comment, "<type_comment>", "func_type")
```
Don't think that is necessary here, not even in strict mode.

Reply:
`Mypy` keeps complaining about it.. I think it might be because of the way we import and redefine `_ast_py3` and it can't follow that?

Reply:
I've been using a dev version of mypy. Seem like this false-positive will be fixed with the next release `0.930`. Let's leave the `type: ignore` in there then.

Reply:
Seems like a number of false positives will be fixed. Last release has been some time ago, I remember we encountered other issues as well in other PRs.

We'll update when they release I guess.